vue/example/app.js

```diff
@@ -0,0 +1,17 @@
+import Vue from 'vue'
+import vueTextMask from '../src/vueTextMask'
+
+Vue.directive('text-mask', vueTextMask)
+
+/* eslint-disable no-new */
+new Vue({
+  el: '#app',
+
+  data: {
+    phone: '',
+
+    maskOptions: {
+      mask: ['(', /[1-9]/, /\d/, /\d/, ')', ' ', /\d/, /\d/, /\d/, '-', /\d/, /\d/, /\d/, /\d/]
+    }
+  }
+})
```

Comment:
It seemed unnecessary as I expected people would know how Vue works. I simplified the documentation to be more directly related to the text mask directive. I have changed it back.